.gitignore

```diff
@@ -0,0 +1 @@
+/.idea
```

Comment:
This should be in your global .gitignore file, not in the project. (https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)